### ISSUE TITLE
Command 'Include What You Use' resulted in an error (command 'extension.iwyu' not found)

### ISSUE THREAD

**Issue by nianyush** [pokowaka/vscode-iwyu#2]
i can run iwyu and python in the workspace correctly. And i have set the iwyu.exe to iwyu. What could be the cause?
![image](https://user-images.githubusercontent.com/5505684/98309656-ffadc380-1f98-11eb-9580-182d7a1a7711.png)

**Comment by pokowaka** [comment]
If possible could you copy the stack trace from the developer window?

**Comment by Hendiadyoin1** [comment]
Same problem with out of the box settings in WSL

**Comment by vvrbanc** [comment]
Hi,

I had the same problem.

The issue was that I did not have the correct path to "compile_commands.json" configured. Attempting to run this extension without that json file results in the mentioned error "command 'extension.iwyu' not found".

To create this file automatically, your project needs to use cmake, and you need to add "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON" when running cmake.

If you don't use cmake, you must create this file manually, the structure looks like this:

```
{
  "directory": "/home/user/src",
  "command": "/usr/lib/llvm/13/bin/clang++ -o /home/user/src/helloworld.cpp.o -c /home/user/src/helloworld.cpp",
  "file": "/home/user/src/helloworld.cpp"
}, ...
```

The "command" must be the full command that is normally ran when compiling that source file with whatever build system you're using.

You need to specify a separate command for every .c/.cpp file of your project that you want to run this extension on.

Basically just copy/paste whatever your build system actually runs for each source file.

**Comment by findNextStep** [comment]
in my case the error disapper after I set my python to python3

**Comment by zowers** [comment]
had same problem because our cmake binary directory is different from default `"${workspaceFolder}/build/compile_commands.json"`.
resolved after changing it in Workspace settings and restarting VSCode
```
"iwyu.compile_commands": "${workspaceFolder}/../build.testconfig/compile_commands.json",
```
